.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
+# Cache dependencies so that they are only recompiled
+# if they or the compiler were upgraded
+cache: cargo
```

Comment:
We want to be careful with enabling caching on Travis. See [this blog post](https://levans.fr/rust_travis_cache.html) about why caching Rust build artifacts may not be desirable; `cargo clean && cargo build && du -sh target/` to calculate the size of `target/` gives me `337M`, it was `1.1G` before I ran `cargo clean`.

Reply:
Great blog post - I've gone ahead and updated the Travis config to include a [`before_cache` command](https://github.com/0xazure/supernova/pull/9/files#diff-354f30a63fb0907d4ad57269548329e3R11) to clear out the remove the redundant directory.